src/main/fc/cli.c

```diff
@@ -1897,7 +1891,7 @@ static void cliVtx(char *cmdline)
             if (ptr) {
                 val = atoi(ptr);
                 // FIXME Use VTX API to get min/max
-                if (val >= VTX_BAND_MIN && val <= VTX_BAND_MAX) {
+                if (val >= VTX_SETTINGS_MIN_BAND && val <= VTX_SETTINGS_MAX_BAND) {
```

Comment:
does the VTX API allow the FIXME here and below to be fixed now?

Reply:
Maybe, but I think that would want to be via a separate PR and testing.